manifests/config.pp

```diff
@@ -164,12 +164,8 @@
     # }
 
     # Generate Elasticsearch config
-    $data = merge(
-      $elasticsearch::config,
-      { 'path.data' => $elasticsearch::datadir },
-      { 'path.logs' => $elasticsearch::logdir },
-      $_tls_config
-    )
+    $data =
+      $elasticsearch::config + { 'path.data' => $elasticsearch::datadir } + { 'path.logs' => $elasticsearch::logdir } + $_tls_config
```

Comment:
i think you could put all of this into one line, but at that point its just nitpicking  :)